### PR TITLE
Add menu items for the plugin settings

### PR DESCRIPTION
**Pull request by zhengbli**
This PR exposes the plugin settings in the "Preferences" menu:
untitled

### CODE REVIEW

File: Main.sublime-menu [hunk 1]
```diff
@@ -1,9 +1,74 @@
...
+      {
+        "id": "package-settings",
+        "children": [
+          {
+            "caption": "TypeScript",
+            "children": [
+              {
+                "command": "open_file",
+                "args": {
+                  "file": "${packages}/TypeScript/Preferences.sublime-settings"
```
Comment (hoanhtien) [inline Main.sublime-menu]: For people cloning the plugin from github the folder name could be TypeScript-Sublime-Plugin?

Comment (zhengbli) [inline Main.sublime-menu]: You are right that is a problem in that case, though there seems to be no way around it.

File: typescript/commands/settings.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import sublime_plugin
+
+from ..libs.global_vars import *
+from ..libs import cli, logger
+import os
+
+class TypescriptOpenPluginDefaultSettingFile(sublime_plugin.WindowCommand):
```
Comment (DanielRosenwasser) [inline typescript/commands/settings.py]: Typescript -> TypeScript
Setting -> Settings

File: typescript/commands/__init__.py [hunk 3]
```diff
@@ -59,5 +64,8 @@
     "TypescriptShowDoc",
     "TypescriptSignaturePanel",
     "TypescriptSignaturePopup",
-    "TypescriptBuildCommand"
+    "TypescriptBuildCommand",
+    "TypescriptOpenPluginDefaultSettingFile",
+    "TypescriptOpenTsDefaultSettingFile",
+    "TypescriptOpenTsreactDefaultSettingFile"
```
Comment (DanielRosenwasser) [inline typescript/commands/__init__.py]: Actually, we can change the casing as part of a separate PR.

File: typescript/commands/settings.py (hunk 2 above)
Comment (zhengbli) [inline typescript/commands/settings.py]: Though Sublime transforms this camel casing to snake casing, which means "TypeScript" will be transformed to "type_script" when referred to in sublime-menu files.

**Pull request merged**